x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/node/styles.tsx

```diff
@@ -38,7 +39,19 @@ export const NODE_HEIGHT = 100;
  * The width of a node label in the graph, in pixels.
  * Must be a multiple of `GRID_SIZE * 2`.
  */
-export const NODE_LABEL_WIDTH = 140;
+export const NODE_TAG_WIDTH = 140;
```

Comment:
Tags are not a node in the graph - this is not relevant for them to be a multiplication of GRID_SIZE*2
And if it is, please add a relevant UT for it (see `constants.test.ts`)